### DISCUSSION TITLE
Document tech decisions not possible

### DISCUSSION THREAD

**Discussion by esrefbozkurt** [DevSkillsHQ/discusssions#63] [Help]
After all the setup hell I was going through unfortunately the file is crazy big now, so that not everything can be loaded. That's why I was not able to document tech decisions. What can we do now?

**Comment by fenske** [comment]
Hi @esrefbozkurt. I'm Anton from Alva. I saw Ludvig assisted you with another ticket you created. I'm sorry to hear you experienced issues with building a binary file. From what I can tell, you could just exclude the node_modules lib from the PR, as it only has generated code that won't impact the code review performed by the hiring team. I've just done it for you, so go ahead and add your review with tech decisions.

Let me know if you have any other questions. 

And I wish you the best of luck with the rest of your interview process!

**Reply by esrefbozkurt** [reply]
Hey @fenske thanks a lot again 🙏🏼. Life saviour. I’m going to continue adding tech decisions soon.
